### PR TITLE
add reindexing option to maintenance agent

### PR DESCRIPTION
**Pull request by shaheemazmalmmd**

### CODE REVIEW

**Comment by bufferoverflow** [comment]
Please rework this and use round brackets as defined here https://github.com/fossology/fossology/blob/master/CONTRIBUTING.md#git-commit-conventions

I think this is a feature so please instead of
```
ui[enhancement] :: added reindexing to maintainance agent
```
use something like
```
feat(maintenance) : add re indexing option
```
as commit message

**Comment by shaheemazmalmmd** [comment]
Sure Will do it . Thank you

**Comment by bufferoverflow** [comment]
Great, just one issue pending, my example had an extra space after the scope, sorry

instead of 
```
feat(maintenance) : add re indexing option
```
it should look like:
```
feat(maintenance): add re indexing option
```

File: src/maintagent/ui/maintagent.php [hunk 1]
```diff
@@ -93,7 +93,8 @@ function DisplayForm()
                      "D"=>_("Vacuum Analyze the database."),
               //       "U"=>_("Process expired uploads (slow)."),
               //       "Z"=>_("Remove orphaned files from the repository (slow)."),
-                     "v"=>_("verbose (turns on debugging output)")
+                     "v"=>_("verbose (turns on debugging output)"),
+                     "I"=>_("Reindexing of database (This activity may take 5-10 mins. Execute only when system is not in use).")
```
Comment (maximilianhuber) [inline src/maintagent/ui/maintagent.php]: Just a minor change: please reorder such that `"v" =>...` is the last item. (after that I will merge this PR)

**Pull request merged**